saleor/settings.py

```diff
@@ -80,7 +80,7 @@ def get_bool_from_env(name, default_value):
     ('uk', _('Ukrainian')),
     ('vi', _('Vietnamese')),
     ('zh-hans', _('Chinese')),
-    ('zh-tw', _('Chinese (Taiwan)'))]
+    ('zh-hant', _('Chinese (Taiwan)'))]
```

Comment:
These two should get renamed to "Chinese (simplified)" and "Chinese (traditional)" respectively.

Reply:
We also need to rename the locale directories, otherwise both will fall back to `zh`.